.github/workflows/builds.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - name: CUDA
             container: ghcr.io/acts-project/ubuntu2004_cuda:v16.1
           - name: SYCL
-            container: ghcr.io/acts-project/ubuntu2004_oneapi:v16.1
+            container: ghcr.io/acts-project/ubuntu2004_oneapi:root
```

Comment:
@beomki-yeo, could you update the PR to `v19`? I think that should work for your changes, and would allow us to finally merge this thing in. :wink:

Reply:
Alright. BTW, #133 should get merged first.

Reply:
hmm it doesn't work...